code/go/internal/validator/semantic/format_checkers.go

```diff
@@ -38,19 +41,28 @@ func (r relativePathChecker) IsFormat(input interface{}) bool {
 	}
 
 	path := filepath.Join(r.currentPath, asString)
-	_, err := fs.Stat(r.fsys, path)
+	info, err := fs.Stat(r.fsys, path)
 	if err != nil {
 		return false
 	}
+
+	// TODO: It happens that we want the same max size for all the files we reference with
```

Comment:
That's true as format checkers are a hackish solution, adapted to our needs. It's a gap in the original parser.